### PR TITLE
Add hpa to sorting order (port to v3)

### PR DESCRIPTION
**Pull request by Constantin07**
**What this PR does / why we need it**:

This is port of #6039 to v3

### CODE REVIEW

File: pkg/releaseutil/kind_sorter_test.go [hunk 1]
```diff
@@ -127,15 +127,19 @@ func TestKindSorter(t *testing.T) {
...
 
 	for _, test := range []struct {
 		description string
 		order       KindSortOrder
 		expected    string
 	}{
-		{"install", InstallOrder, "abcde1fgh2ijklmnopqrstuvw!"},
-		{"uninstall", UninstallOrder, "wvmutsrqponlkji2hgf1edcba!"},
+		{"install", InstallOrder, "abcde1fgh2ijklmnopqrxstuvw!"},
+		{"uninstall", UninstallOrder, "wvmutsrrqponlkji2hgf1edcba!"},
```
Comment (hickeyma) [inline pkg/releaseutil/kind_sorter_test.go]: wvmutsrrqponlkji2hgf1edcba! --> wvmutsxrqponlkji2hgf1edcba!

**Comment by bacongobbler** [comment]
Would you mind fixing the failing tests?

> pkg/releaseutil/kind_sorter_test.go:132:11: undefined: util

**Review by hickeyma — CHANGES_REQUESTED** [review]
@Constantin07 Thanks for the PR. Small issue with the test.

File: pkg/releaseutil/kind_sorter_test.go (hunk 1 above)
Comment (Constantin07) [reply pkg/releaseutil/kind_sorter_test.go]: Apologies, my bad.

**Comment by Constantin07** [comment]
Corrected now.

**Review by hickeyma — APPROVED** [review]
@Constantin07 LGTM, thanks

**Pull request merged**